src/index.js

```diff
@@ -51,7 +51,7 @@ Object.assign(Lock.prototype, {
    * Acquire a lock for a specific ID value. Callback returns the following value:
    *
    *   {
-   *     id: Message ID
+   *     id: id of lock
    *     success: either true (lock was acquired) of false (lock was not aquired)
    *     index: modification index for current lock
```

Comment:
Please remove `id` and `index` from the docs now since the client doesn't need to use those.